lib/player.rb

```diff
@@ -0,0 +1,6 @@
+class Player
+  attr_accessor :player_pick
+  def player_pick
+    ["rock", "paper", "scissors"].sample
```

Comment:
Not sure you would need to randomise the selection for the human too as this input by them but I see why you have done so for the spec.